relay-filter/src/common.rs

```diff
@@ -62,13 +66,16 @@ impl FilterStatKey {
             FilterStatKey::WebCrawlers => "web-crawlers",
             FilterStatKey::InvalidCsp => "invalid-csp",
             FilterStatKey::FilteredTransactions => "filtered-transaction",
-        }
+            FilterStatKey::GenericFilter(filter_name) => {
```

Comment:
nit: instead of looking at it as a "name", we could look at it as an identifier. Today this is a name, but tomorrow could be a UUID.